src/checkFirebaseSDKVersion.ts

```diff
@@ -58,7 +59,13 @@ export function checkFunctionsSDKVersion(options: any): void {
     return;
   }
 
-  const sourceDir = path.join(options.config.projectDir, options.config.get("functions.source"));
+  const sourceDirName = options.config.get("functions.source");
+  if (!sourceDirName) {
```

Comment:
I'm really confused here. How does this check actually verify that the path exists or not? How does this fall back to the directory "functions"?